test/test-debugletapi.js

```diff
@@ -206,7 +206,7 @@ describe('Debuglet API', function() {
     });
 
     it('should not tie down event loop on large breakpoints', function (done) {
```

Comment:
This test appears to test the time it takes to stringify the object. If it doesn't test our application logic can we get rid of it?